tcms/signals.py

```diff
@@ -209,19 +209,33 @@ def handle_comments_pre_delete(sender, **kwargs):
 def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
     """
     Send email updates to assignee after they've been
-    assigned a bug on bug creation.
+    assigned a bug on bug creation or there is a new comment to the bug.
```

Comment:
this doc-string is already out of date and doesn't match the actual behavior of this function. IMO behavior can be understood by reading the source code so we don't really need the doc-string unless pylint requires it. 

Either update the text to match behavior or make it more generic/simple so that there isn't such mismatch at all. (Or remove it if the tools are happy).